plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/validators/N4JSExpressionValidator.xtend

```diff
@@ -168,6 +169,23 @@ class N4JSExpressionValidator extends AbstractN4JSDeclarativeValidator {
 	override register(EValidatorRegistrar registrar) {
 		// nop
 	}
+	
+	
+	/** Disallow use of elements without runtime representation as values. */
+	@Check
+	def checkPureTypeAsValue(IdentifierRef idRef) {
```

Comment:
Wouldn't the name `#checkHollowTypeAsValue()` be more in line with the terminology used in other places?